src/components/Chip.tsx

```diff
@@ -1,23 +1,36 @@
 import React from "react";
 import { usePresentationContext } from "src/components/PresentationContext";
-import { Css, Margin, Only, Xss } from "src/Css";
+import { Css, Margin, Only, Properties, Xss } from "src/Css";
 import { useTestIds } from "src/utils/useTestIds";
 
+type ChipType = "caution" | "warning" | "success" | "neutral";
+
+// exporting for using in type prop as constant - this could be moved and become a global list for colors
+export const ChipTypes: Record<ChipType, ChipType> = {
```

Comment:
@stephenh - I exposed the `ChipTypes` Record so we can use it for consistency in our implementation values, I noticed people are manually typing in 'success` etc for the `Tag` component in blueprint.